js/cronsync.js

```diff
@@ -250,11 +252,11 @@ CronSync.prototype = {
     // sync if it is sufficiently recent.
 
     // - Initiate a sync of the folder covering the desired time range.
-    this._LOG.syncAccount_begin(account.id);
-    this._LOG.syncAccountHeaders_begin(account.id, null);
+    logic(scope, 'syncAccount_begin');
```

Comment:
No change required.  Discussion point:  The _begin/_end stuff was part of the automatically generated names for asyncJobs where identifier rules limited the choices.  While I'm not sure the "foo:begin" "foo:end" stuff is the best naming convention either, I do like it more than underscores.  Do you think colon works for convention and automation, or do you favor something else?